checker_protocol.md

```diff
@@ -85,9 +87,12 @@ The method to be executed in this task.
 `havoc`:
 * A `havoc` MUST NOT depend on any previous tasks or credentials that should have been created during a previous task.
 * A `havoc` MAY use e.g. credentials from previous tasks, provided it does not fail if these credentials are missing.
+* A `havoc` MAY be called at any time during a round, even before `putflag` and `putnoise` of the same round.
```

Comment:
This mixes stating facts and requirements, maybe
 
`havoc` MUST be prepared being to be called before `putflag` and `putnoise` for the current `roundId`

Reply:
I wouldn't call it mixing facts and requirements exactly, since this specifies the interface between the caller (engine) and checker, we might also impose requirements on the caller in this specification. But on the other hand, we can also just rephrase that

Reply:
Thoughts on the reworded spec? It does not mention that explicitly, but I think the first rule should be sufficiently explicit. We also have a general rule mentioning that checkers `MUST NOT make any assumptions about the order of the execution of any tasks`

Reply:
Sounds good.